### PR TITLE
docs: fix default and description for empty_seed_starts_cluster property

### LINKED ISSUE #1049: empty_seed_starts_cluster default should be `true`
### Describe the Issue

https://docs.redpanda.com/docs/reference/node-properties/#empty_seed_starts_cluster this documents that:

>While false by default for backward compatibility, empty_seed_starts_cluster is recommended to be set to true.

This appears reversed: the default is true: https://github.com/redpanda-data/redpanda/blob/dev/src/v/config/node_config.cc#L75 and *true* maintains backwards compatible behaviour. `False` would be what we'd recommend, too.

### PR DESCRIPTION
**Pull request by shorgi**
Fixes #1049 .

Review deadline: Fri Jan 20.

### CODE REVIEW

File: docs/reference/node-properties.mdx [hunk 1]
```diff
@@ -109,11 +109,13 @@ Flag to enable developer mode, which skips most of the checks performed at start
 
 Controls how a new cluster is formed. This property must have the same value in all nodes in a cluster.
 
+If `true`, an empty [seed_servers](#seed_servers) list will denote that this node should form a cluster. At most one node in the cluster should be configured with an empty seed_servers list. If no such configured node exists, or if configured to `false`, all nodes denoted by the seed_servers list must be identical among those nodes' configurations, and those nodes will form the initial cluster.
```
Comment (Feediver1) [inline docs/reference/node-properties.mdx:112]:
```suggestion
If `true`, an empty [seed_servers](#seed_servers) list will denote that this node should form a cluster. At most, one node in the cluster should be configured with an empty seed_servers list. If no such configured node exists, or if configured to be `false`, all nodes denoted by the seed_servers list must be identical among those nodes' configurations, and those nodes will form the initial cluster.
```

File: docs/reference/node-properties.mdx [hunk 2]
```diff
@@ -109,11 +109,13 @@ Flag to enable developer mode, which skips most of the checks performed at start
...
+
 :::tip
-While `false` by default for backward compatibility, `empty_seed_starts_cluster` is recommended to be set to `true`.
+While it's set `true` by default for backward compatibility, empty_seed_starts_cluster is recommended to be set `false`.
```
Comment (Feediver1) [inline docs/reference/node-properties.mdx:115]:
```suggestion
While it's set `true` by default for backward compatibility, Redpanda recommends that you set empty_seed_starts_cluster to `false`.
```

**Review by Feediver1 — APPROVED** [review]
See suggested edits. Your call.

File: docs/reference/node-properties.mdx [hunk 3]
```diff
@@ -109,11 +109,13 @@ Flag to enable developer mode, which skips most of the checks performed at start
 
 Controls how a new cluster is formed. This property must have the same value in all nodes in a cluster.
 
+If `true`, an empty [seed_servers](#seed_servers) list will denote that this node should form a cluster. At most, one node in the cluster should be configured with an empty seed_servers list. If no such configured node exists, or if configured to be `false`, all nodes denoted by the seed_servers list must be identical among those nodes' configurations, and those nodes will form the initial cluster.
+
 :::tip
-While `false` by default for backward compatibility, `empty_seed_starts_cluster` is recommended to be set to `true`.
+While it's set `true` by default for backward compatibility, Redpanda recommends that you set empty_seed_starts_cluster to `false`.
```
Comment (JFlath) [inline docs/reference/node-properties.mdx:115]: Nitpick: Should this the prop name have backticks?

```suggestion
While it's set `true` by default for backward compatibility, Redpanda recommends that you set `empty_seed_starts_cluster` to `false`.
```

**Review by JFlath — APPROVED** [review]

**Pull request merged**